Makefile

```diff
@@ -82,7 +82,7 @@ include $(NXDK_DIR)/lib/Makefile
 OBJS = $(addsuffix .obj, $(basename $(SRCS)))
 
 ifneq ($(NXDK_CXX),)
-include $(NXDK_DIR)/lib/libcxx/Makefile.nxdk
+include $(NXDK_DIR)/lib/libcxx/Makefile.nxdk 
```

Comment:
would you mind removing that whitespace?

Reply:
I sure would not, it was never supposed to be there in the first place.